DocumentService/DocumentService.csproj

```diff
@@ -19,4 +19,8 @@
 	    <Version>2.6.0</Version>
 	  </PackageReference>
 	</ItemGroup>
+
+	<ItemGroup>
+	  <PackageReference Include="DocumentFormat.OpenXml" Version="2.19.0" />
+	</ItemGroup>
```

Comment:
- Inconsistency in how `PackageReference` tag is opened and closed. Use the self-closing `PackageReference` tab.

Reply:
Updated project file to have self closing `PackageReference` tags. Fixed the indentation in project file.